xscen/spatial.py

```diff
@@ -119,6 +123,7 @@ def _dot(arr, wei):
     )
 
 
+@parse_config
```

Comment:
I'm not sure it's a good idea to add parse_config to this function because it is called by other functions that might not use it in the same way.
Somebody might use `subset` in one way in their workflow and need to use it in a different way in `spatial_mean` , but not realise that the config is used in both cases...